db/schema.rb

```diff
@@ -269,6 +271,7 @@
     t.integer "published_by_id"
     t.integer "parent_id"
     t.boolean "preferred"
+    t.date "omb_approval_date"
```

Comment:
Where is the migration file that generates this change? Doesn't appear to be in the PR